### ISSUE TITLE
More examples of defined risks/causes and effects

### ISSUE THREAD

**Issue by agorlov** [yegor256/0rsk#42]
It would be easier to start using the tool if there were more examples from real projects.

I would be thankful.

**Comment by yegor256** [comment]
@agorlov good idea, let me try to create something like a simple "risk taxonomy"

**Comment by yegor256** [comment]
@agorlov check this out: https://github.com/yegor256/awesome-risks

**Comment by agorlov** [comment]
Good point to start.
Thank you for discovering something new for me.

**Comment by agorlov** [comment]
May be, put link to  https://github.com/yegor256/awesome-risks to form help block:
![image](https://user-images.githubusercontent.com/2485728/58550482-7d9f6400-8216-11e9-9c76-b5bb94a0514e.png)

or to footer

**Comment by 0crat** [comment]
@agorlov Thanks for suggesting an enhancement! You've earned +18 points. If you find anything else in the repository that [doesn't look](https://www.yegor256.com/2018/02/06/where-to-find-more-bugs.html) as good as you might expect, [do not hesitate](https://www.yegor256.com/2014/04/13/bugs-are-welcome.html) to [report](https://www.yegor256.com/2018/04/24/right-way-to-report-bugs.html) bugs too. Your running score is [+18](https://zerocracy.github.io/judges-action/zerocracy-vitals.html); don't forget to check your [Zerocracy account](https://www.zerocracy.com/) too).
